problems/P026/assignment-26.py

```diff
@@ -0,0 +1,84 @@
+def cal(stack):
+    i=-1
+    t=0
+    print(stack)
+    while i >-len(stack):
+      if stack[i]=="*":
+        stack[i+1]*=stack[i-1]
```

Comment:
You can assign array members to variables before conditions and check on variables.

It makes code more readable and also easier to be changed in the future.